tests/integration/fileTypeTests.py

```diff
@@ -29,57 +30,25 @@ def setUpClass(cls):
-        s3FileNameAwardValidShortcols = cls.uploadFile("awardValidShortcols.csv", user)
 
         # Create submissions and get IDs back
         submissionIDs = {}
-        for i in range(0, 16):
-            if i == 7:
-               # Award financial mixed will be second quarter
-                submissionIDs[i] = cls.insertSubmission(cls.jobTracker, user, datetime(2015,3,15))
-            else:
-                submissionIDs[i] = cls.insertSubmission(cls.jobTracker, user)
+        for i in range(0, 9):
+            submissionIDs[i] = cls.insertSubmission(cls.jobTracker, user)
 
 
         # Create jobs
         jobDb = cls.jobTracker
         statusReady = str(jobDb.getJobStatusId("ready"))
         jobTypeCsv = str(jobDb.getJobTypeId("csv_record_validation"))
-        jobTypeValidation = str(jobDb.getJobTypeId("validation"))
         jobInfoList = {
             "valid": [statusReady, jobTypeCsv, str(submissionIDs[1]), s3FileNameValid, 1],
-            "mixed": [statusReady, jobTypeCsv, str(submissionIDs[2]), s3FileNameMixed, 1],
             "programValid": [statusReady, jobTypeCsv, str(submissionIDs[4]), s3FileNameProgramValid, 2],
-            "programMixed": [statusReady, jobTypeCsv, str(submissionIDs[5]), s3FileNameProgramMixed, 2],
             "awardFinValid": [statusReady, jobTypeCsv, str(submissionIDs[6]), s3FileNameAwardFinValid, 3],
-            "awardFinMixed": [statusReady, jobTypeCsv, str(submissionIDs[7]), s3FileNameAwardFinMixed, 3],
-            "awardValid": [statusReady, jobTypeCsv, str(submissionIDs[8]), s3FileNameAwardValid, 4],
-            "awardMixed": [statusReady, jobTypeCsv, str(submissionIDs[9]), s3FileNameAwardMixed, 4],
-            "awardMixedDelimiter": [statusReady, jobTypeCsv, str(submissionIDs[10]), s3FileNameAwardMixedDelimiter, 4],
-            "crossApprop": [statusReady, jobTypeCsv, str(submissionIDs[11]), s3FileNameCrossApprop, 1],
-            "crossPgmAct": [statusReady, jobTypeCsv, str(submissionIDs[11]), s3FileNameCrossPgmAct, 2],
-            "crossAwardFin": [statusReady, jobTypeCsv, str(submissionIDs[11]), s3FileNameCrossAwardFin, 3],
-            "crossAward": [statusReady, jobTypeCsv, str(submissionIDs[11]), s3FileNameCrossAward, 4],
-            "crossFile": [statusReady, jobTypeValidation, str(submissionIDs[11]), None, None],
-            "appropValidShortcols": [statusReady, jobTypeCsv, str(submissionIDs[12]), s3FileNameAppropValidShortcols, 1],
-            "programMixedShortcols": [statusReady, jobTypeCsv, str(submissionIDs[13]), s3FileNameProgramMixedShortcols, 2],
-            "awardFinMixedShortcols": [statusReady, jobTypeCsv, str(submissionIDs[14]), s3FileNameAwardFinMixedShortcols, 3],
-            "awardValidShortcols": [statusReady, jobTypeCsv, str(submissionIDs[15]), s3FileNameAwardValidShortcols, 4]
+            "awardValid": [statusReady, jobTypeCsv, str(submissionIDs[8]), s3FileNameAwardValid, 4]
```

Comment:
I know this is legacy code, but as long we're cleaning up our tests, it would be great to replace the file type "magic numbers" in the `jobInfoList` dict with a lookup from the `file_type` table.